osbenchmark/builder/launchers/launcher.py

```diff
@@ -15,12 +15,13 @@ def start(self, host, node_configurations):
         """
         raise NotImplementedError
 
-    def stop(self, host, nodes):
+    def stop(self, host, nodes, metrics_store):
```

Comment:
This interface looks confusing to me. As a user of this interface I'm wondering the interface/class asks me to provide metrics_store only for the stop method?

Is there a reason why we shouldn't have the metrics_store as a class level instance. My understanding is that it can't vary with time - so once this class is instantiated, the metrics_store can just live with it all through its lifetime.

Reply:
`metrics_store` being a parameter always struck me as odd as well. I like the idea to add it to the constructor, will refactor